openedx/core/djangoapps/credit/urls.py

```diff
@@ -4,11 +4,17 @@
 from django.conf.urls import patterns, url
 
 from .views import create_credit_request, credit_provider_callback
+from .api.provider import get_credit_provider_info
 
 urlpatterns = patterns(
     '',
 
     url(
+        r"^v1/provider/(?P<provider_id>[^/]+)/$",
```

Comment:
@aamir-khan the changes for api url patterns has been merged please update this url accordingly:
```python
r"^v1/providers/(?P<provider_id>[^/]+)/$",
```